### DISCUSSION TITLE
Vaultwarden does not start after latest upgrade

### DISCUSSION THREAD

**Discussion by pianoquintet** [dani-garcia/vaultwarden#2863] [Q&A]
Hello,

I just recreated the docker image to upgrade to latest version on an ARM debian machine (was running the immediately previous one) but the system now fails to start.  Portainer pops up this error when I try to start it :"Failure. Request failed with status code 400"

If I inspect the container, this is what is shown under State:

State
Dead false
Error failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: exec: "/usr/bin/dumb-init": stat /usr/bin/dumb-init: no such file or directory: unknown
ExitCode 127
FinishedAt 0001-01-01T00:00:00Z
OOMKilled false
Paused false
Pid 0
Restarting false
Running false
StartedAt 0001-01-01T00:00:00Z
Status created

Please help.

**Comment by BlackDex** [comment]
See #2826
